example/screens/RoomScreen.tsx

```diff
@@ -91,10 +97,13 @@ const RoomScreen = ({ navigation }: Props) => {
 
   return (
     <SafeAreaView style={styles.container}>
-      {tracks.length > 0 ? (
+      {tracks.length > 0 && !isIosSimulator ? (
```

Comment:
```suggestion
      {tracks.length > 0 ? (
```
For video grid simulator condition is not necessary. It will display other peers video track.